integration-tests/jms-artemis-client/src/main/resources/application.properties

```diff
@@ -0,0 +1,17 @@
+## ---------------------------------------------------------------------------
+## Licensed to the Apache Software Foundation (ASF) under one or more
+## contributor license agreements.  See the NOTICE file distributed with
+## this work for additional information regarding copyright ownership.
+## The ASF licenses this file to You under the Apache License, Version 2.0
+## (the "License"); you may not use this file except in compliance with
+## the License.  You may obtain a copy of the License at
+##
+##      http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## ---------------------------------------------------------------------------
+quarkus.artemis.enabled=true
```

Comment:
It would be great to have a hint why quarkus.artemis.enabled=true is here. The following is my guess, please fix it as needed:

```suggestion

# Overridden to false via @TestProfile(JmsArtemisDisable.class) for some tests
# When false, we produce a custom ConnectionFactory via CustomConnectionFactory producer bean
quarkus.artemis.enabled=true
```

Reply:
It looks good to me, Thanks!